main/openvswitch/APKBUILD

```diff
@@ -19,11 +19,10 @@ source="http://openvswitch.org/releases/$pkgname-$pkgver.tar.gz
 	ovs-vswitchd.confd
 	ovs-modules.initd
 	musl-if_packet.patch
-	0002-fix-wait-any.patch
 	ifupdown-alpine.patch
-	readme.debian.patch
-	libressl-compat.patch
+	README.alpine
```

Comment:
https://github.com/alpinelinux/aports/pull/3771#discussion_r178434194

Reply:
reverted to how the PR originally was & rebased again